OlinkAnalyze/tests/testthat/test-Olink_anova.R

```diff
@@ -15,9 +15,16 @@ extension_control <- npx_data_format221010_no_NA |>
   dplyr::mutate(Assay = gsub("Incubation", "Extension", Assay)) |>
   dplyr::mutate(UniProt = gsub("INC", "EXT", UniProt))
 
+# Add dummy CTRL assay using the data for ACY3
+assay_ctrl <- npx_data_format221010_no_NA |>
+  dplyr::filter(stringr::str_detect(Assay, "ACY3")) |>
+  dplyr::mutate(Assay = gsub("ACY3", "CTRL", Assay)) |>
+  dplyr::mutate(UniProt = gsub("Q96HD9", "P40313", UniProt))
+
 npx_data_format221010_ext_ctrl <- rbind(
```

Comment:
can we add a test that has assay_ctrl but not extension_control and check that no error is thrown?

Reply:
Added this to unit tests in new commit, confirmed that no error is thrown.